ddtrace/contrib/internal/openai/_endpoint_hooks.py

```diff
@@ -717,3 +717,41 @@ def _record_response(self, pin, integration, span, args, kwargs, resp, error):
         else:
             span.set_metric("openai.response.total_bytes", getattr(resp, "total_bytes", 0))
         return resp
+
+
+class _ResponseHook(_BaseCompletionHook):
+    _request_arg_params = ("api_key", "api_base", "api_type", "request_id", "api_version", "organization")
```

Comment:
```suggestion
    _request_arg_params = ()
```
These args above aren't present in the responses create() API. We shouldn't need to worry about these